pkgs/steam-im-modules/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, extra-cmake-modules
+, wrapGAppsHook3
+, wrapGAppsHook4
+, libsForQt5
+, gtk3
+, gtk4
+, lib
+}:
+stdenv.mkDerivation {
+  pname = "steam-im-modules";
+  version = "jupiter-20240131";
+
+  src = fetchFromGitHub {
+    owner = "valve-project";
+    repo = "steam-qt-keyboard-plugin";
+    rev = "eab7d4c13bbe1372b7b28731b8209c12037e91af";
+    sha256 = "L9rKEUxDT0fYrvnPIMJS5/wSq1W2KowreaxeXmcCq60=";
+  };
+
+  nativeBuildInputs = [
+    extra-cmake-modules
+    wrapGAppsHook3
```

Comment:
I don't think any of the hooks should be necessary, given it's just a library.

Reply:
Yes, you are right. I should check docs first. I'll remove them later.